ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -366,7 +366,7 @@ pub(crate) enum Instruction {
         args: Vec<Operand>,
     },
     #[deku(id = "5")]
-    Br,
+    Br { succ: BBlockIdx },
```

Comment:
Can we document what `succ` is? I know what it is now, but I guarantee I'll forget soon :)